docs/usage.md

```diff
@@ -89,27 +130,140 @@ You can also supply a run name to resume a specific run: `-resume [run-name]`. U
 
 ```nextflow
 process {
-  withName: star {
-    memory = 32.GB
-  }
+    withName: STAR_ALIGN {
+        memory = 100.GB
+    }
 }
 ```
 
-See the main [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for more information.
+> **NB:** We specify just the process name i.e. `STAR_ALIGN` in the config file and not the full task name string that is printed to screen in the error message or on the terminal whilst the pipeline is running i.e. `RNASEQ:ALIGN_STAR:STAR_ALIGN`. You may get a warning suggesting that the process selector isn't recognised but you can ignore that if the process name has been specified correctly. This is something that needs to be fixed upstream in core Nextflow.
+
+### Tool-specific options
+
+For the ultimate flexibility, we have implemented and are using Nextflow DSL2 modules in a way where it is possible for both developers and users to change tool-specific command-line arguments (e.g. providing an additional command-line argument to the `STAR_ALIGN` process) as well as publishing options (e.g. saving files produced by the `STAR_ALIGN` process that aren't saved by default by the pipeline). In the majority of instances, as a user you won't have to change the default options set by the pipeline developer(s), however, there may be edge cases where creating a simple custom config file can improve the behaviour of the pipeline if for example it is failing due to a weird error that requires setting a tool-specific parameter to deal with smaller / larger genomes.
```

Comment:
Might need some generalization here - this is likely from viralrecon / rnaseq?

Reply:
An updated usage.md is present in the latest commit 079018b1c9ab5a4cec4824d9ab0ff5bde77dffc8